docs/adr/0008-kommunikasjon-mellom-sbs-og-regel-api.md

```diff
@@ -0,0 +1,25 @@
+# Kommunikasjon mellom applikasjon i sbs og regel-api 
+
+* Status: pending
+* Deciders: Team Dagpenger
+* Date: 2019-10-06 
+
+## Context and Problem Statement
+
+Vår nye dagpengekalkulator er en front-end som skal leve i sbs. Den trenger å snakke med dp-regel-api i fss.
+I fremtiden vil vi også få flere ting som lever i sbs, så vi må finne en fornuftig måte å kommunisere 
+
+## Considered Options
+
+* Sonekryssing 
+* Deploye regel-api i begge soner
+* Bruke kafka som regel-api-endepunkt 
+* … 
+
+## Pros and Cons of the Options 
+
+### Sonekryssing 
```

Comment:
Vært lettere å diskutere hvis vi legger til noen pros og cons for hver av disse. 


Sonekryssing: 
Pros: 
 - en gangs jobb? 
cons: 
- Uklart hvem som vi må hjelp av? 
- Må lage proxy? 

Deploye regel-api i begge soner
Pros:
- Kan forholde seg til samme API 
Cons:
- Mere å monitorere
- Kobling til database må åpnes (er åpnet?)

Bruke kafka som regel-api-endepunkt 
Pros: 
- 
Cons:
- Må duplisere logikk som allerede er implementert i dp-regel-api
- "Dingsen" som leser til og fra kafka og returnerer resultater må ha state (ref over egentlig)

Reply:
> * Kobling til database må åpnes (er åpnet?)

Ja, PostgreSQL er ikke delt i soner (FSS/SBS)

Reply:
Når det gjeldet deploy av regel api i sbs så  er kobling til database åpnet. Det som trengs er vault konfig slik at poddene i sbs kan få tak i credentials.

Reply:
Jeg resolver også skravler vi per punkt :)